lib/trino-parquet/src/main/java/io/trino/parquet/DataPage.java

```diff
@@ -15,8 +15,9 @@
 
 import java.util.OptionalLong;
 
-public abstract class DataPage
+public abstract sealed class DataPage
```

Comment:
Are those allowed in Trino already?

Reply:
yup, a new avenue of commit creation is available ;)